mridefacer

```diff
@@ -308,9 +382,26 @@ for infile in "$@"; do
   # make explicit test for success, as imcp doesn't
   [ "`$FSLDIR/bin/imtest "$outbase"_defacemask`" = "1" ]
 
-  if [ "$md_apply2orig" = 1 ]; then
+  if [ "$md_apply2orig" = 1 ] || [ "$md_apply" = 1 ]; then
     $FSLDIR/bin/fslmaths "$infile" -mul "$wdir/out_${count}_defacemask" "$wdir/out_${count}_defaced" -odt input
-    $FSLDIR/bin/immv "$wdir/out_${count}_defaced" "${outbase}"
+    if [ "$md_apply2orig" = 1 ]; then
+      outsuffix=''
+    else
+      outsuffix='_defaced'
+    fi
+
+    if [ $md_apply = 1 ] && [ $md_keep_orig = 1 ]; then
+      orig_fname=$(echo $infile | sed -e 's,\.nii.*,_orig,g')
+      $FSLDIR/bin/immv "$infile" "${orig_fname}"
+      [ $under_annex = 0 ] || (
+        cd "$infile_dir"; git add $(/bin/ls $(basename ${orig_fname})*);
```

Comment:
This could be a single command: `git -C $infile_dir add ...`

Reply:
did it but then reverted back and removed use of  `ls` to get the files and rely on globbing (wouldn't be possible without cd since would try to complete `*` in prev dir)... anyway RFing to avoid all the checks all the time